pyaerocom/io/cams2_83/reader.py

```diff
@@ -443,6 +458,11 @@ def read_var(self, var_name: str, ts_type: str | None = None, **kwargs) -> Gridd
     data_id = "CAMS2-83.EMEP.day0.AN"
     reader = ReadCAMS2_83(data_dir=data_dir, data_id=data_id)
     reader.daterange = ("2021-12-01", "2021-12-04")
+    print(
```

Comment:
Shouldn't this go in a `logger` call?

Reply:
These are some smaller convenient self-tests under `if __name__ == "__main__":`
Only used during development and without logger at this stage.